### DISCUSSION TITLE
gotta ask before i ruin something with flashing spi

### DISCUSSION THREAD

**Discussion by unevoljitelj** [Joshua-Riek/ubuntu-rockchip#942] [General]
opi5 was on a shelf for more then 6 months, now i have found some use for it but it has spi flash from first build of debian made for it. dunno if this is a problem, some posts on reddit say it might be. so i want to reflash it with newer stuff.
before i break anything i just wanted to check with someone

sudo u-boot-install-mtd /dev/mtdblock0 

this will flash newer/ubuntu boot loader?
do i have to clear spi first and what would be command to do it?
it will boot from sd first but also from nvme if sd is not present?  armbian would also boot or it requures to reflash it again?

thanks!

**Comment by FreeWilly2022** [comment]
Yes, the u-boot-install-mtd will write over whatever is there. When I check out the distros on Armbian I only run them from the SD card, and they work fine with JR's u-boot.

**Reply by unevoljitelj** [reply]
Thank you!
